src/predict/api/prediction.py

```
@@ -405,14 +406,14 @@ def predictions_for_heatmap():
-            response["error"] = "No data found."
-            status_code = 404
-    except Exception as e:
-        response["error"] = f"Unfortunately an error occured"
-        current_app.logger.error("Error: ", str(e), exc_info=True)
-        status_code = 500
-    finally:
-        return jsonify(response), status_code
+            response["error"] = "No data found."
+            status_code = 404
+    except Exception as e:
+        response["error"] = f"Unfortunately an error occured"
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)